src/web/index.js

```diff
@@ -123,6 +125,36 @@ render(
   document.getElementById('root')
 );
 
+/**
+ * We display a loading indicator on startup that overlays the rest of the app.
```

Comment:
This is probably over-engineered. But, just using a simple timer always ended up being too long or too short. The timer I ended up with just kind of bounces along until state changes settle down and the app stabilizes.

Reply:
But, totally willing to yank this chunk out and just drop in a straight timeout for the loader. Figured we could iterate on it along with however we want to finesse the loader component itself.

Reply:
I agree that this is hacky and overengineered. Can we prospectively be more precise than just hoping the state stops changing? Perhaps looking for specific values to be non-`null`?